docs/example.rst

```diff
@@ -1,97 +1,58 @@
 
-When you have selected a number of cells, you can add them to the dataset
-by clicking “Add”. Click “Save” when you are done.
+.. image:: images/train-4.png
 
-.. image:: images/test-3.png
+Clicking a slide in the gallery will load this slide in the heatmap viewer, where the user can identify regions for annotation. Clicking ``Show Segmentation`` will display the heatmap overlay on the slide viewer, and the user can zoom to hotspots to provide corrections to the classifier. At high-magnification, objects are displayed with color-coded boundaries to indicate their predicted class (green = positive). Prediction errors can be corrected directly in the slide viewer by double clicking an object to cycle it's class, adding this object to the training set. The classifier can be retrained with the ``Retrain`` button.
 
+.. image:: images/train-5.png
 
+.. note:: Object labels can be cycled in the heatmap menu by double-click. Objects that have been annotated and added to the training set will appear with yellow boundaries. An object can be removed from the training set by double-clicking a second time.
 
+When the training is completed, click the ``Finalize`` button in the ``Instance`` interface to save the training set to the database. This training set can be reloaded and resumed from using the *Continue a session* option on the main page.
 
-Report with training dataset
+Reviewing a training set
 ------------------------------
+Annotations in a validation set can be reviewed using the review interface.
 
-HistomicsML provides a reporting tool that enables users to download their training results.
-
-Go to http://localhost/HistomicsML/reports.html?application=nuclei
-
-.. image:: images/report-1.png
+At the home page under *Continue a session*, select the dataset and training set name and click ``Continue``. Navigate to the ``Review`` interface by clicking the tab at the top menu.
 
-* Slide class counts: counts (numbers) of samples that fall into different categories. For example, the numbers of positive and negative samples in a slide. (.csv)
-* Map scores: score, centroid X, and centroid Y for each sample on a slide. (.csv)
-* Download a training set: a training set. (.h5)
-* Apply classifier: scores for each prediction. Predicted scores will be added to the current dataset. (.h5)
+The review interface displays the annotated objects organized by class and slide. Thumbnail images of the objects are organized into columns by class. Clicking a thumbnail will bring that object into the field of view in the slide view. The thumbnails can be dragged/dropped to a different column to change the class label, or placed in the ``Ignore`` column to discard them from the set. Changes are instantly commited to the database (no additional button clicks are needed).
```

Comment:
@slee172 Are changes made in Review by dragging thumbnails instantly committed to the database? Do we need to use retrain (for training) or save (for validation) to commit these changes?

Reply:
@cooperlab Changes are not going to the database directly. Instead, they stay on the memory of the learner server. Once the users finalize their training on the instance tab, all the training samples are stored in the database.